### PR TITLE
Multiple UI changes

### PR DESCRIPTION
**Pull request by asaadmahmoodspin**
Hey guys have a look. Changes include.
- Fixed gifs on smaller screens
- Fixing sidebar bugs on safari (Reverting it to position absolute, now it won't scroll on safari along with the settings content)
- Improving UI for team settings.
- Fixing tutorial stuff, it was off centered maybe due to some centre channel changes and also making it a bit more compatible with other themes.
- Changing the text for search when no results are found.

### CODE REVIEW

File: web/react/components/tutorial/tutorial_intro_screens.jsx [hunk 1]
```diff
@@ -35,6 +35,11 @@ export default class TutorialIntroScreens extends React.Component {
         preference = PreferenceStore.setPreference(Preferences.TUTORIAL_STEP, UserStore.getCurrentId(), newValue);
         AsyncClient.savePreferences([preference]);
     }
+    componentDidMount() {
+        const height = $(window).height() - 100;
+        $('.tutorials__scroll').height(height);
```
Comment (jwilander) [inline web/react/components/tutorial/tutorial_intro_screens.jsx]: We should do this in the render function like this:
```
const height = $(window).height() - 100;
```
and
```
<div className='tutorials__scroll' style={{height}}>
    //stuff
</div>
```

**Comment by jwilander** [comment]
+1 other than my one comment

File: web/react/components/settings_sidebar.jsx [hunk 2]
```diff
@@ -5,6 +5,9 @@ export default class SettingsSidebar extends React.Component {
     componentDidUpdate() {
         $('.settings-modal').find('.modal-body').scrollTop(0);
         $('.settings-modal').find('.modal-body').perfectScrollbar('update');
+        if (navigator.userAgent.indexOf('Safari') !== -1 && navigator.userAgent.indexOf('Chrome') === -1) {
```
Comment (crspeller) [inline web/react/components/settings_sidebar.jsx]: These should be functions in utils.

**Comment by asaadmahmoodspin** [comment]
Done. Please review.

**Pull request merged**